source/doc-pages/introduction.md

```diff
@@ -1,35 +1,62 @@
-Ruby Object Mapper (ROM) is a Ruby library with the goal to provide powerful object
-mapping capabilities without limiting the full power of your datastore. ROM is based
-on several concepts and decisions that differentiate it from your normal Ruby ORM:
+Ruby Object Mapper (ROM) is a Ruby persistence library with the goal to provide 
```

Comment:
The original intent to persistence was to be inclusive to the plethora of adapters (eg. csv) which isn't (to me) a database per se. I'm not really sure what people in the industry at large call this stuff. We at Tenjin call it the persistence layer, hence the choice here.